packages/api3-voting/contracts/Api3Voting.sol

```diff
@@ -265,9 +261,9 @@ contract Api3Voting is IForwarder, AragonApp {
         userAddressToLastNewProposalTimestamp[msg.sender] = now;
 
         uint64 snapshotBlock = getBlockNumber64() - 1; // avoid double voting in this very block
-        uint256 votingPower = token.totalSupplyAt(snapshotBlock);
```

Comment:
You should have kept `totalSupplyAt` here because (total voting power) == (total number of pool shares) and (1 staked token) != (1 pool share) after any reward or claim payouts.